bsc.verilog/derived_bits/derived_bits.exp

```diff
@@ -0,0 +1,20 @@
+compile_verilog_pass DerivedBits.bs "" "-opt-undetermined-vals"
```

Comment:
Might there be value in testing the behavior again, but without the `-opt-undetermined-vals` flag?